app/models/authentication_token.rb

```diff
@@ -5,16 +5,17 @@
 # Table name: authentication_tokens
 #
 #  id           :bigint           not null, primary key
-#  enabled      :boolean          default(TRUE)
+#  enabled      :boolean          default(TRUE), not null
```

Comment:
do these changes belong in this PR?

Reply:
no, will rewrite these away...

Reply:
although it does look like `schema.rb` is out of step with the migration so we'll have to fix this up elsewhere - will raise a separate PR perhaps...